lametro/templates/event/_event_header_live_public_comment.html

```diff
@@ -53,78 +53,41 @@
         {% endif %}
     </section>
 
-    {% if has_agenda and not event.has_passed %}
+    {% if has_agenda and event_notices %}
```

Comment:
I'm wondering now if it would be easier to get the appropriate notice/s in the view so we can simplify the templates?

Reply:
I think it would be easier once we figure out how to get those template variables to work within these notices. As of now, the conditionals that are left within these three comment templates have event specific variables in them that we wouldn't currently be able to render in the editable notice. Unless you had something else in mind!

Reply:
Got it, ok. Let's put a pin in this.